frontend/src/pages/AdminPanel.js

```diff
@@ -0,0 +1,1253 @@
+                  </div>
+                ))}
+                <button
+                  type="button"
+                  onClick={() => setLocalMeats([...localMeats, ""])}
+                  className="mt-2 px-4 py-2 bg-blue-500 text-white rounded hover:bg-blue-600"
+                >
+                  Dodaj mięso
+                </button>
+              </div>
+
+              {/* Zmiana statusu */}
+                <div className="mb-4">
+                  <label className="font-bold block">Status kebaba:</label>
+                  <select
+                    value={localStatus}
+                    onChange={(e) => setSelectedKebab((prev) => ({ ...prev, status: e.target.value }))}
+                    className="w-full px-4 py-2 border rounded"
+                  >
+                    <option value="exists">Istnieje</option>
+                    <option value="closed">Zamknięty</option>
+                    <option value="planned">Planowany</option>
+                  </select>
+                </div>
+
+              {/* Godziny otwarcia */}
+              <div className="mb-4">
+                <label className="font-bold block">Godziny otwarcia:</label>
+                {Object.entries(localOpeningHours).map(([day, hours], index) => (
+                  <div key={index} className="flex items-center space-x-2 mb-2">
+                    <span className="w-1/4">{day}</span>
+                    <input
+                      type="text"
+                      value={hours}
+                      onChange={(e) => {
+                        const updatedHours = { ...localOpeningHours, [day]: e.target.value };
+                        setLocalOpeningHours(updatedHours);
+                      }}
+                      className="w-3/4 px-2 py-1 border rounded"
+                    />
+                    <button
+                      onClick={() => {
+                        const updatedHours = { ...localOpeningHours };
+                        if (day in updatedHours) delete updatedHours[day];
+                        setLocalOpeningHours(updatedHours);
+                      }}
+                      className="text-red-500"
+                    >
+                      Usuń
+                    </button>
+                  </div>
+                ))}
+                <button
+                  onClick={() => {
+                    const days = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday'];
+                    const availableDay = days.find((day) => day in localOpeningHours && !localOpeningHours[day]);
+                    if (availableDay) {
+                      setLocalOpeningHours({ ...localOpeningHours, [availableDay]: '' });
+                    } else {
+                      alert('Wszystkie dni są już dodane.');
+                    }
+                  }}
+                  className="text-green-500"
+                >
+                  Dodaj dzień
+                </button>
```

Comment:
I think it's bugged. Deleting a day works well, but If you want to add a day there's an error
![image](https://github.com/user-attachments/assets/92afdd1d-1002-4168-b19d-a4e19377c3d9)

Reply:
changed ✔️